great-idea/css/index.css

```diff
@@ -63,4 +63,87 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.container {
+	width: 880px;
+	margin: 0 auto;
+}
+
+.navbar {
+	border: 1px solid red;
+}
+
+.navigation .logo {
+	display: flex;
+	justify-content: flex-end;
+}
```

Comment:
since this is just referring to one element with no children, you don't need to display: flex on this one. flex specifies how children of the element should appear